docs/tutorials/smab_mo_cc.ipynb

```diff
@@ -20,7 +20,7 @@
     "from rich import print\n",
     "\n",
     "from pybandits.model import Beta, BetaMOCC\n",
-    "from pybandits.smab import SmabBernoulliMOCC, create_smab_bernoulli_mo_cc_cold_start"
+    "from pybandits.smab import SmabBernoulliMOCC"
    ]
   },
   {
```

Comment:
in cell [3] `n_objectives = 2` must not be specified. The number of objective is directly deduced by the shape of the Betas list.